<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="ZChartView" name="chartView" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAction">
    <property name="title">
     <string>Action</string>
    </property>
    <addaction name="actionZoom"/>
    <addaction name="actionReset_Zoom"/>
   </widget>
   <addaction name="menuAction"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSelection_Mode"/>
   <addaction name="separator"/>
   <addaction name="actionHorizontal_Zoom"/>
   <addaction name="actionZoom"/>
   <addaction name="actionReset_Zoom"/>
   <addaction name="actionPan"/>
   <addaction name="separator"/>
   <addaction name="actionlegend"/>
  </widget>
  <action name="actionReset_Zoom">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/spettro1.xpm</normaloff>:/images/spettro1.xpm</iconset>
   </property>
   <property name="text">
    <string>Reset Zoom</string>
   </property>
  </action>
  <action name="actionZoom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/zoom_patt.xpm</normaloff>:/images/zoom_patt.xpm</iconset>
   </property>
   <property name="text">
    <string>Zoom</string>
   </property>
  </action>
  <action name="actionSelection_Mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/selectoff.png</normaloff>:/images/selectoff.png</iconset>
   </property>
   <property name="text">
    <string>Selection Mode</string>
   </property>
   <property name="toolTip">
    <string>Selection Mode</string>
   </property>
  </action>
  <action name="actionPan">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/pan.png</normaloff>:/images/pan.png</iconset>
   </property>
   <property name="text">
    <string>Pan</string>
   </property>
   <property name="toolTip">
    <string>Pan</string>
   </property>
  </action>
  <action name="actionHorizontal_Zoom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/horizontal_zoom.png</normaloff>:/images/horizontal_zoom.png</iconset>
   </property>
   <property name="text">
    <string>Horizontal Zoom</string>
   </property>
  </action>
  <action name="actionlegend">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/legendaon.xpm</normaloff>
     <normalon>:/images/legendaoff.xpm</normalon>:/images/legendaon.xpm</iconset>
   </property>
   <property name="text">
    <string>legend</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ZChartView</class>
   <extends>QWidget</extends>
   <header>zchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
